src/libs/s3api/s3_object_ops.py

```diff
@@ -273,7 +273,7 @@ def delete_s3_objects(self, bucket_name, object_prefix=None):
             objects = []
             for file in self.get_boto3_resource().Bucket(bucket_name).objects.all():
                 if file.key.startswith(object_prefix):
-                    objects.append({"key": file.key})
```

Comment:
Please, take latest pull as issue fixed and verified in https://github.com/Seagate/corio/pull/66

Reply:
ok